sdk/storage/storage-file/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2019.11 12.0.0-preview.6
+
+- Bug Fix - Previous versions of `@azure/storage-file` library failed for the react-apps because of the usage of `fs.stat` method which is not available in browsers. The issue is fixed in this new release.
```

Comment:
> `@azure/storage-file` [](start = 32, length = 22)

why it's file instead of blob?

Reply:
As I've mentioned in the description already, this PR includes the changes from #5762. Once that is merged, this change won't show up here.

And you're right, I still need to update changelog for blob.

Reply:
Added changelog for blob.